src/index.scss

```diff
@@ -0,0 +1,17 @@
+@import './commonStyles/reset.scss';
+@import './commonStyles/variables.scss';
+@import './Footer/Footer.scss';
+
+body {
```

Comment:
nope. this way styles below will overwrite styles above. move it to separate file `base-styles.scss` and import in here. However, this styles aren't necessary